src/Reflection/InitializerExprTypeResolver.php

```diff
@@ -1468,9 +1469,6 @@ private function callOperatorTypeSpecifyingExtensions(Expr\BinaryOp $expr, Type
 		return null;
 	}
 
-	/**
-	 * @param BinaryOp\Plus|BinaryOp\Minus|BinaryOp\Mul|BinaryOp\Div $expr
```

Comment:
Please add ShiftLeft and ShiftRight here, instead of removing the PHPDoc.

Reply:
Done for both. This should lead to a unreachable statement for the "else" branch, not sure if that is a problem.

Reply:
> This should lead to

It doesn't in bleeding edge. PHPStan no longer penalizes for unreachable else branch that throws an exception.